doc/user/content/sql/create-cluster.md

```diff
@@ -105,16 +105,18 @@ _replica_name_ | A name for a cluster replica.
 
 {{% replica-options %}}
 
-{{< note >}}
-If you do not specify an availability zone, Materialize will automatically
-assign the availability zone with the least existing replicas for the
-associated cluster to increase the cluster's tolerance to availability zone
-failure.
-
-To check the availability zone associated with each replica in a cluster, use
-the [`mz_cluster_replicas`](/sql/system-catalog/mz_catalog/#mz_cluster_replicas)
-system table.
-{{< /note >}}
+## Availability Zones and Scheduling
+
+When scheduling replicas, Materialize makes the following guarantees, within each cluster:
```

Comment:
```suggestion
When assigning replicas of a managed cluster to availability zones, Materialize makes the following guarantees within each cluster:

{{< warning >}}
These guarantees to do not apply to unmanaged clusters.
{{< /warning >}}
```

(There are _some_ guarantees, but I think it's too confusing to try to talk about them.)

Reply:
they do apply to unmanaged clusters though!